src/components/global/TextInput.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 function TextInput(props) {
-  const { label, textValue, placeholderValue, onChangeHandler } = props;
+  const { label, textValue, placeholderValue, onChangeHandler, typeValue } = props;
```

Comment:
The TextInput component is meant to already have a type prop, I feel like I've PRed it into main like 4 times but for some reason it still isn't there. My current PR implements a ```type``` prop though, which will conflict with this if it's merged. 
I've also added a default to the "text" type if no prop is given. If it's okay, could you remove your changes to this file, and change your uses of the TextInput component to use ```type``` instead of ```typeValue?```

Reply:
I've just merged that PR, so these changes should be made please

Reply:
yea just removed my changed from TextInput